eng/Publishing.props

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <PropertyGroup>
+    <PublishingVersion>3</PublishingVersion>
+    <PublishDependsOnTargets>$(PublishDependsOnTargets);_PublishPackages</PublishDependsOnTargets>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <_PackagesToPublish Remove="@(_PackagesToPublish)" />
+    <!--
+      Additional packages needed for source-only VMR build - https://github.com/dotnet/source-build/issues/4205
+    -->
+    <_PackagesToPublish Include="$(ArtifactsPackagesDir)Release\*.nupkg" UploadPathSegment="Runtime" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
+  </ItemGroup>
+
+  <Target Name="_PublishPackages">
```

Comment:
I'm not sure why this additional target is needed at all. Can you elaborate?

Reply:
Arcade publishing infra only collects packages under `$(ArtifactsPackagesDir)/Shipping` and `$(ArtifactsPackagesDir)/NonShipping`. Additional packages are present under `$(ArtifactsPackagesDir)/Reelease` that are needed in VMR build. We get package downgrade issue in `format` repo. I tried to investigate this and couldn't figure out why flowing the live versions is not enough in this case. I have created a tracking issue: https://github.com/dotnet/source-build/issues/4205

Reply:
Release and Shipping actually contain the same packages just with differing branding. Arcade only publishes the non-stable versions today and I think that is correct. I suspect that there is something else going on with the package downgrade.

Anyways, why would the target be required to achieve this?

Reply:
See my same comment in the fsharp PR. This doesn't need a target and should just be this: https://github.com/dotnet/fsharp/pull/16838#discussion_r1520097757

Reply:
OK - I'll remove the target and use the simpler model for both repos.

Reply:
Fixed with https://github.com/dotnet/symreader/pull/312/commits/37ea36f12c35f127cdcad33c27e9de8d02ea02d6